docs/source/user/settings.rst

```diff
@@ -200,6 +200,38 @@ This setting can be used to set the maximum size limit for firmware images, eg:
     The numeric value represents the size of files in bytes.
     Setting this to ``None`` will mean there's no max size.
 
+
+``OPENWISP_RADIUS_CALLED_STATION_ID``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Default**: ``{}``
+
+This setting should contain information of OpenVPN Management Interface for fetching
```

Comment:
This setting allows to specify the parameters to connect to the different OpenVPN management interfaces available for an organization.

Or something along these lines.